src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -56,6 +56,7 @@ public DissolutionPatchResponse addDirectorApproval(String companyNumber, String
         if (!this.hasDirectorsLeftToApprove(dissolution)) {
             setDissolutionStatus(dissolution, ApplicationStatus.PENDING_PAYMENT);
             dissolution.setCertificate(this.certificateGenerator.generateDissolutionCertificate(dissolution));
+            dissolutionEmailService.sendPendingPaymentEmail(dissolution);
```

Comment:
Could we move these 3 lines into a private method i.e. `handleFinalApproval(dissolution);` to keep the public method as light as possible